code.js

```diff
@@ -7,11 +7,11 @@ if (a == 0) {
 }
 
 if (a == 1) {
-    return arr[1]; 
+    return arr; 
 } 
 else { 
     for (let i = 2; i <= a; i++) { 
```

Comment:
According to your code, It does not seem like you are using any recursive call. You need to use at least one recursive call.